app/javascript/packages/address-search/types.d.ts

```diff
@@ -24,7 +23,6 @@ interface PostOffice {
   weekday_hours: string;
   zip_code_4: string;
   zip_code_5: string;
-  is_pilot: boolean;
 }
```

Comment:
One other thought I had- do you think we should version up for address-search (app/javascript/packages/address-search/package.json) because we are changing the interface? If you agree- we should also update `app/javascript/packages/address-search/CHANGELOG.md`

Reply:
I agree, that's a good idea. 👍🏻 I will get to that tomorrow.

Reply:
If getting access will take a bit of time, add some time on my calendar. In that session, you can merge this in and I can publish it for you. Ideal we should do it back to back so we know there is no drift.  (I want to make sure I have access to publish before you merge it in.)

Reply:
Thanks, Gina!  I booked time on your calendar for tomorrow.